.gitignore

```diff
@@ -236,7 +237,7 @@ Temporary Items
 
 # VisualStudioCode
 .vscode/*
-# !.vscode/settings.json
+!.vscode/settings.json
```

Comment:
Damn. I really didn't want to add vscode settings. This is the most compelling case I've seen though.

Mainly it's not going to help non-vscode users. So I'd like to try to be inclusive of them. Is there a file we can add that will configure the behavior of the ts language server instead? Then that will work for all contributors.

Can we leave the vscode config directory out of this PR. Instead add it to the dev container config (see for example https://github.com/microsoft/vscode-dev-containers/blob/main/containers/go/.devcontainer/devcontainer.json#L19).

Reply:
took it out [ee59445](https://github.com/seamapi/react/pull/464/commits/ee594451bc510218302268b8f222656dc90a7d96)

Reply:
dev container: [96324e1](https://github.com/seamapi/react/pull/464/commits/96324e1cc50f73f660e14be5c8663e1533dc4de7)